### PR TITLE
[ACCEL] Modify course maping with subsection prerequisite

### PR DESCRIPTION
**Pull request by Golub-Sergey**
Description: Modify course maping with subsection prerequisite. Depends on role staff.

Youtrack: [ticket](https://youtrack.raccoongang.com/issue/ACCEL-309)

Testing instructions:

1. Open course in studio and make subsection with prerequisites
2. Go to lms course like staff or superuser
3. Change role of viewing course from staff to leaner

### CODE REVIEW

File: lms/templates/course_experience/course-outline-fragment.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+course_sections = blocks.get('children')
+%>
+
+<%
+    staff = None
+    if 'masquerade_settings' in request.session:
+        course_key = CourseKey.from_string(request.session['course_id']) if request.session['masquerade_settings'] else None
+
+        if not request.session['masquerade_settings'] and not course_key:
+            staff = 'staff'
```
Comment (flying-pi) [inline lms/templates/course_experience/course-outline-fragment.html]: Could you change the name of this var? It is a little bit confusing when staff is not staff.

File: lms/templates/course_experience/course-outline-fragment.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+%>
+
+<%
+course_sections = blocks.get('children')
+%>
+
+<%
+    staff = None
+    if 'masquerade_settings' in request.session:
+        course_key = CourseKey.from_string(request.session['course_id']) if request.session['masquerade_settings'] else None
```
Comment (flying-pi) [inline lms/templates/course_experience/course-outline-fragment.html]: Move condition to the top if and remove ternary operator. 

I.E. 
```python
course_key=None
if 'masquerade_settings' in request.session and request.session['masquerade_settings']:
        course_key = CourseKey.from_string(request.session['course_id'])
```

**Review by flying-pi — CHANGES_REQUESTED** [review]

File: lms/templates/course_experience/course-outline-fragment.html (hunk 1 above)
Comment (Golub-Sergey) [reply lms/templates/course_experience/course-outline-fragment.html]: done

File: lms/templates/course_experience/course-outline-fragment.html (hunk 2 above)
Comment (Golub-Sergey) [reply lms/templates/course_experience/course-outline-fragment.html]: done

**Review by flying-pi — APPROVED** [review]

**Pull request merged**